readers-frontend/src/components/DesktopNavbar.astro

```diff
@@ -103,6 +128,33 @@ else computedBaseColour = "#000";
       </button>
     </li>
   </ul>
+  {subMenuItems && <ul
```

Comment:
I initially didn't design it to be aligned just because the subsection may have a much larger name than the sections above. Anyhow I'll try it with a grid and see If it works

Reply:
<img width="771" alt="Screenshot 2023-05-10 at 8 23 28 pm" src="https://github.com/yorknouse/website/assets/58376513/fe459cdd-b939-4ac9-ab14-e4c3d5be8f96">
So as I expected on smaller screens it breaks with longer subsections

Reply:
Yeah fair enough. I don't know about without the grid tho. With the submenus starting so much earlier than the categories it looks like some bad css

Reply:
Yeah you're right, I'm adding more padding to fix it